docs/advanced/docker-reference.md

```diff
@@ -0,0 +1,204 @@
+# Docker Command Quick Reference
+
+This list may not be comprehensive, but we hope it's a good start for those new
+to docker and Open ONI.
+
+- [Rebuild Dev Containers](#rebuild-dev-containers)
+- [Manage Data](#manage-data)
+- [Run Commands from Containers](#run-commands-from-containers)
+- [Django Reference](#django-reference)
+- [Manage Containers, Volumes, and Imges](#manage-containers-volumes-and-images)
+
+## Rebuild Dev Containers
+
+Builds or rebuilds the development setup.  Useful if you changed something
+copied into the container, such as the various helper scripts in the `docker`
+directory:
+
+```bash
+docker-compose down
+docker-compose up
+```
+
+Or if you don't want to be "trapped" in the log output: `docker-compose up -d`
+
+Remember, you can always get at logs via `docker-compose logs`, and drill down
+just to the logs you want, which is usually the web service: `docker-compose logs web`.
+
+## Manage Data
+
+### Load a Batch
+
+You will need to put NDNP-compliant newspaper batches in the `data/batches`
+directory at the root of the project.  You can either wget a batch from
+Chronicling America like below, or you can put some of our [short sample
+batches](https://github.com/open-oni/sample-data) into the data directory.
+
+Mirroring a batch from Chronicling America is easy, but can take a *lot* of time:
```

Comment:
To avoid duped documentation, this could link to the other page about getting batches (`docs/manage-data/batches-load-purge.md#obtain-batches`) - or have that page link here